### PR TITLE
Add initialisation data 

### PR DESCRIPTION
**Pull request by tswinb**
Add initialisation data for IrisVOs, Capacity, and Allocations

### CODE REVIEW

File: iris-initialisation.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+INSERT INTO IrisVOs VALUES
+    ("lsst"),("lz"),("skatelescope.eu"),("dune"),("vo.cta.in2p3.fr"),
```
Comment (tofu-rocketry) [inline iris-initialisation.sql]: What happens with the VOs that have both capitalised and non-capitalised versions?

Comment (tswinb) [reply iris-initialisation.sql]: Only one shows in the filtering list but both show on the graph.

Comment (tofu-rocketry) [reply iris-initialisation.sql]: Ah, so the actual application of filtering isn't case sensitive, right?

Comment (tswinb) [reply iris-initialisation.sql]: Right

**Pull request merged**